docker/web/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+MAINTAINER Pedro Marques <roque@juniper.net>
+RUN apt-get install -y software-properties-common
+RUN apt-add-repository -y ppa:anantha-l/opencontrail-2.20
+RUN apt-get update
+RUN apt-get -y install gdebi wget
+
+RUN wget -qO /root/nodejs_0.8.15-1contrail1_amd64.deb https://launchpad.net/~opencontrail/+archive/ubuntu/ppa/+files/nodejs_0.8.15-1contrail1_amd64.deb
+RUN gdebi -n /root/nodejs_0.8.15-1contrail1_amd64.deb
```

Comment:
you are downloading the package manually vs using apt-get download; that doesn't seem required.
And then using an extra tool for the install of the dependent package... what problem does it solve ?